pkg/scaffold/scaffold.go

```diff
@@ -53,7 +53,7 @@ type Scaffold struct {
 	BoilerplateOptional bool
 
 	// Project is the project
-	Project input.ProjectFile
+	Project *input.ProjectFile
 
 	ProjectOptional bool
```

Comment:
When is used the ProjectOptional?

Reply:
Same places as above. The usage is to skip errors when trying to load the file if it doesnt exist. IIRC it is used for some tests as the Project and Boilerplate files do not really exist.